### PR TITLE
Docs fix spelling issues

### PR DESCRIPTION
**Pull request by nnsW3**
Hello
I and fixed several spelling issues in your docs.
hope it helps,
Br, Elias.

### CODE REVIEW

**Comment by DN6** [comment]
cc: @stevhliu

File: docs/source/en/tutorials/tutorial_overview.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ specific language governing permissions and limitations under the License.
 
 # Overview
 
-Welcome to 🧨 Diffusers! If you're new to diffusion models and generative AI, and want to learn more, then you've come to the right place. These beginner-friendly tutorials are designed to provide a gentle introduction to diffusion models and help you understand the library fundamentals - the core components and how 🧨 Diffusers is meant to be used.
+Welcome to 🧨 Diffusers! If you're new to diffusion models and generative AI, and want to learn more, then you've come to the right place. These beginner-friendly tutorials are designed to provide a gentle introduction to diffusion models and help you understand the library fundamentals - the core components and how 🧨 Diffusers are meant to be used.
```
Comment (stevhliu) [inline docs/source/en/tutorials/tutorial_overview.md:15]: This doesn't need to be changed since "Diffusers" is singular.

**Review by stevhliu — COMMENTED** [review]
Thanks for the improvements!

**Comment by nnsW3** [comment]
done

**Pull request merged**